cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -187,8 +188,12 @@ func generateUUID(bd blockdevice.BlockDevice) (string, bool) {
 		uuidField = bd.PartitionInfo.PartitionEntryUUID
 		ok = true
 	case len(bd.DeviceAttributes.WWN) > 0:
-		klog.Infof("device(%s) has a WWN, using WWN: %s", bd.DevPath, bd.DeviceAttributes.WWN)
-		uuidField = bd.DeviceAttributes.WWN
+		// if device has WWN, both WWN and Serial will be used for UUID generation
```

Comment:
Add another comment here on why serial is also used.

Reply:
Done.